dts/msm8916/msm8916-mtp.dts

```diff
@@ -7,6 +7,10 @@
 / {
 	compatible = "qcom,msm8916-mtp", "qcom,msm8916";
 
+	// The bootloader of UZ801 requires a dts with this borad-id to boot
```

Comment:
Do we want to keep this given the stock aboot seems to be dubious anyway? Maybe just make lk1st mandatory for the board and drop this?

Reply:
Please keep it, it's still useful for backing up stock partitions without entering QDL (by temporarily booting to lk2nd)